tmp/lanelet2_extension/lib/utilities.cpp

```diff
@@ -213,6 +213,9 @@ lanelet::LineString3d getLineStringFromArcLength(
   lanelet::Points3d points;
   double accumulated_length = 0;
   size_t start_index = linestring.size();
+  if (start_index == 0) {
+    return lanelet::LineString3d{lanelet::InvalId, points};
+  }
```

Comment:
Can I ask for more detail about how the node side will behave with this change?
I'm curious about whether the node side will work well with lanelet::Invalid.

Reply:
I didn't check how the node behaves actually. But at least this function always return `lanelet::LineString3d{lanelet::InvalId, points};`.